test/DGmethods/courant.jl

```diff
@@ -0,0 +1,149 @@
+using Test
+using MPI
+using CLIMA
+using CLIMA.Mesh.Topologies
+using CLIMA.Mesh.Grids
+using CLIMA.Mesh.Grids: VerticalDirection, HorizontalDirection, EveryDirection
+using CLIMA.VTK
+using Logging
+using Printf
+using LinearAlgebra
+using CLIMA.DGmethods: DGModel, init_ode_state, LocalGeometry, courant
+using CLIMA.DGmethods.NumericalFluxes: Rusanov, CentralGradPenalty,
+                                       CentralNumericalFluxDiffusive
+
+using CLIMA.PlanetParameters: kappa_d
+using CLIMA.Atmos: AtmosModel,
+                   AtmosAcousticLinearModel, RemainderModel,
+                   NoOrientation,
+                   NoReferenceState, ReferenceState,
+                   DryModel, NoRadiation, NoSubsidence, PeriodicBC,
+                   Gravity, HydrostaticState, IsothermalProfile,
+                   ConstantViscosityWithDivergence, vars_state, soundspeed
+
+const ArrayType = CLIMA.array_type()
+
+using CLIMA.MoistThermodynamics: air_density, total_energy, internal_energy,
+                                 soundspeed_air
+
+using CLIMA.VariableTemplates: Vars
+using StaticArrays
+
+function initialcondition!(state, aux, coords, t)
+  FT = eltype(state)
+
+  p∞::FT = 10 ^ 5
+  T∞::FT = 300
+  translation_speed::FT = 150
+  translation_angle::FT = pi / 4
+  α = translation_angle
+  u∞ = SVector(translation_speed * cos(α), translation_speed * sin(α), 0)
+
+  u = u∞
+  T = T∞
+  # adiabatic/isentropic relation
+  p = p∞ * (T / T∞) ^ (FT(1) / kappa_d)
+  ρ = air_density(T, p)
+
+  state.ρ = ρ
+  state.ρu = ρ * u
+  e_kin = u' * u / 2
+  state.ρe = ρ * total_energy(e_kin, FT(0), T)
+
+  nothing
+end
+
+
+let
+  # boiler plate MPI stuff
+  CLIMA.init()
+  mpicomm = MPI.COMM_WORLD
+  ll = uppercase(get(ENV, "JULIA_LOG_LEVEL", "INFO"))
+  loglevel = ll == "DEBUG" ? Logging.Debug :
+  ll == "WARN"  ? Logging.Warn  :
+  ll == "ERROR" ? Logging.Error : Logging.Info
+  logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
+  global_logger(ConsoleLogger(logger_stream, loglevel))
```

Comment:
[`CLIMA.init()`](https://github.com/climate-machine/CLIMA/blob/master/src/Driver/Driver.jl#L133) now does all this.